src/main/kotlin/com/dansplugins/fiefs/command/fiefs/FiefsCommand.kt

```diff
@@ -37,11 +45,13 @@ class FiefsCommand(private val plugin: Fiefs) : CommandExecutor, TabCompleter {
         command: Command,
         alias: String,
         args: Array<out String>
-    ): List<String> {
+    ): List<String>? {
```

Comment:
In what cases does this return null?

Reply:
I'm still a bit fuzzy on the tab completer stuff. It's possible this wasn't intentional.

Reply:
This is now getting cast as a List<String> when returning, so it shouldn't ever be able to be null.

Reply:
I've switched over to this:
```
    override fun onTabComplete(
        sender: CommandSender,
        command: Command,
        alias: String,
        args: Array<out String>
    ): List<String> {
        return emptyList()
    }
```

How does that look?

Reply:
Much better, thanks!